### ISSUE TITLE
rsocket / websocket how to set max frame payload size

### ISSUE THREAD

**Issue by srinivasvsk** [spring-projects/spring-framework#25696]
The documentation is not clear how to set the max frame payload size, all questions are pointing to cloud gateway. Currently we are not using gateway and just using rsocker server.
Thanks

**Comment by rstoyanchev** [comment]
@philwebb I don't think this relates entirely to Spring Boot where the RSocket server is configured and started.

@srinivasvsk I'm not sure if your goal is to fragment large RSocket Payloads, or to limit the frame size at the WebSocket transport level? For the former you can set `RSocketServer#fragment` via `RSocketServerCustomizer`. For the latter you'd need to customize RSocket's `WebsocketServerTransport`. In the upcoming RSocket 1.1 there is a [convenient method](https://github.com/rsocket/rsocket-java/blob/6959390a02e598d52135c10f6dfb84c6e3323d6f/rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/server/BaseWebsocketServerTransport.java#L44) to do this but in RSocket 1.0.x you'd have to override the `start` method. In either case, however I don't see any option in Boot to customize the transport which is created privately in Boot, in `NettyRSocketServerFactory`.

**Comment by srinivasvsk** [comment]
Thanks for more details, my use case is to pass a large payload, process and stream results back. Had the server complained about 6500 limit.  can you point to more information on RSocketServerCustomizer
